pkg/controller/ippool/defaults.go

```diff
@@ -203,6 +203,11 @@ func fillDefaults(ctx context.Context, client client.Client, instance *operator.
 			}
 		}
 
+		if pool.DisableNewAllocations == nil {
+			disabled := false
```

Comment:
If you're interested, I did make a convenience method in the ptr package to get points without having to allocate a non pointer variable (and it's generic) `ptr.ToPtr(false)` would return a pointer to a false value.